### ISSUE TITLE
Breaks should be time based rather than round based

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#209]
Internally breaks and rounds are two variants of the same thing (`Round`). That largely works well, but it causes some problems. 

The biggest problem is that when round duration is changed (currently by speed factor, but that itself will be replaced by something better), the breaks remain in between the levels that are in the original structure.  So if you want to play a structure that started out with 20 minute rounds with a break every 5 rounds (e.g. after 100 minutes of play) and you speed it up by a factor of 10, your break now comes after 10 minutes of play, which probably isn't what you want.

Internally there's another problem, because certain things that happen after a particular level has finished (e.g. lammer redemption, end of rebuy period) are specified by a zero-based number that represents how many `Round`s have elapsed. That's error prone and it also means that the value can't directly be told to humans (without misleading them!), which is why the lobby currently doesn't report those values. It's not that translating them would be hard, it's that I've known I want a better solution for a while, but couldn't be bothered to write up this issue...

**Comment by ctm** [comment]
Chris M. mentioned that PS schedules their breaks to start at 55 minutes after the hour and last five minutes. They don't pause the hand in the middle though. Instead the break starts at the end of the last relevant hand.

Seems to me I should provide that as an option, although I don't think I want to make it the only option.

On one hand I don't want to make things too complex, on the other hand Rust has my back.

Realistically, two things make for hands taking a long time: [discarding via typing sucks](https://github.com/ctm/mb2-doc/issues/211) in many ways and there is no decent feedback concerning whose turn it is (no issue yet, but I know that's a big deal). As such, it probably makes sense to do those two things and then revisit this ... unless player feedback says otherwise.

**Comment by ctm** [comment]
Sabyl has explicitly asked for this.

**Comment by ctm** [comment]
I am temporarily making this a high priority, because Rust really does have my back. I should be able to make an option to where breaks are done on the 55 and also make that be the default without a lot of effort.  

Giving everyone 5 minutes from the end of the last hand wouldn't be too hard either, but I'm not going to do that soon, in part because mb2 allows people to call time indefinitely. That policy itself allows people to be a nuisance to the rest of their table, but w/o feedback it could allow a person to be a nuisance to the entire game in a way that wouldn't necessarily be obvious to people on other tables.

**Comment by ctm** [comment]
I think we're going to need this for the heads up tournament (#406), so I've made it a FARGO Prerequisite.

**Comment by ctm** [comment]
I made a bunch of progress on this yesterday. With luck, I'll be able to deploy it today.

**Comment by ctm** [comment]
I deployed this for yesterday's night game. It seemed to work. I still have a few nits to clean up, but I've merged the code into master.

- [ ] Double check to make sure that time spent in a sync break doesn't shorten the level it occurs in ...
- [ ] Check to see that the right thing happens if a sync-break tournament is scheduled in the break window
- [ ] Make sure that demo *DOES NOT* used synchronized breaks because then it will be a mystery sandwich if people start in (or just before) the 55.
- [ ] I added some warn! calls to `tournament.rs` that lack context.  Add context
